proxy/src/main/java/com/wavefront/agent/PushAgent.java

```diff
@@ -112,12 +112,15 @@ protected void startListeners() {
         if (strPort.trim().length() > 0) {
           try {
             int port = Integer.parseInt(strPort);
+            if (discardPointsHours == 0){
+              discardPointsHours = 8760; // number of hours in a year
```

Comment:
make the static variable public so that you can reference constants in a single place